arangod/IResearch/IResearchView.cpp

```diff
@@ -731,10 +700,9 @@ IResearchView::Snapshot const* IResearchView::snapshot(
   TRI_ASSERT(ctx);
   try {
     // collect snapshots from all requested links
-    auto iterate = [&]<typename T>(T const& collections) -> Snapshot const* {
+    auto iterate = [&](auto const& collections, auto h) -> Snapshot const* {
```

Comment:
Why not leave  template here?

Reply:
Because we have the same code in 3.9 (C++17 only) and it is better if we change the abseil map to another type, then now nothing changes, otherwise, old code depends on collections type, ~~also it more beaty~~